client/src/FiltersPage.js

```diff
@@ -0,0 +1,181 @@
+import './App.css';
+import Login from './Login';
+import CardComponent from './CardComponent';
+import Filters from './Filters';
+import NavBar from './navbar';
+import React from 'react';
+import './Filter.css';
+
+export default function FiltersPage() {
+  ///////////////////////////////////////////////////////////////////////////////////
+  ///////////////////////////////////////////////////////////////////////////////////
+  // Map and List view mode buttons
+
+  const [mode, setMode] = React.useState(1);
+
+  function render() {
+    if (mode == 1) {
+      return (
+        <div>
+          <CardComponent
+            name="Bianca's Radishes"
+            location="Berkeley, CA"
+            tags={['vegetables', 'fruit']}
+          />
+          <CardComponent
+            name="Bianca's Radishes"
+            location="Berkeley, CA"
+            tags={['vegetables', 'fruit']}
+          />
+          <CardComponent
+            name="Bianca's Radishes"
+            location="Berkeley, CA"
+            tags={['vegetables', 'fruit']}
+          />
+          <CardComponent
+            name="Bianca's Radishes"
+            location="Berkeley, CA"
+            tags={['vegetables', 'fruit']}
+          />
+          <CardComponent
+            name="Bianca's Radishes"
+            location="Berkeley, CA"
+            tags={['vegetables', 'fruit']}
+          />
+          <CardComponent
+            name="Bianca's Radishes"
+            location="Berkeley, CA"
+            tags={['vegetables', 'fruit']}
+          />
+          <CardComponent
+            name="Bianca's Radishes"
+            location="Berkeley, CA"
+            tags={['vegetables', 'fruit']}
+          />
+        </div>
+      );
+    } else {
+      return <div>map please </div>;
+    }
+  }
+
+  function list_button() {
+    if (mode == 1) {
+      return (
```

Comment:
Let's try to make this a bit more concise and make the className a variable so it'd look something like this:

```javascript
const isActive = mode == 1 ? "activeButton" : "inactiveButton";
return (
    <button
        className={isActive}
        type="button"
        onClick={() => setMode(1)}
    >
        List
    </button>
);
```